test/resources/stubs/testprojekt-ets6-functions.json

```diff
@@ -0,0 +1,129 @@
+            "description": null,
+            "lines": {
+                "0": {
+                    "name": "",
+                    "description": null,
+                    "devices": [],
+                    "medium_type": "KNXnet/IP (IP)"
+                }
+            }
+        },
+        "1": {
+            "name": "",
+            "description": null,
+            "lines": {
+                "0": {
+                    "name": "",
+                    "description": null,
+                    "devices": [],
+                    "medium_type": "KNXnet/IP (IP)"
+                },
+                "1": {
+                    "name": "",
+                    "description": null,
+                    "devices": [],
+                    "medium_type": "Twisted Pair (TP)"
+                }
+            }
+        }
+    },
+    "devices": {},
+    "group_addresses": {
+        "0/0/1": {
+            "name": "Schalten",
+            "identifier": "GA-1",
+            "raw_address": 1,
+            "address": "0/0/1",
+            "project_uid": 14,
+            "dpt": {
+                "main": 1,
+                "sub": 1
+            },
+            "communication_object_ids": [],
+            "description": "Livingroom LivingroomLight"
+        },
+        "0/0/2": {
+            "name": "Status",
+            "identifier": "GA-2",
+            "raw_address": 2,
+            "address": "0/0/2",
+            "project_uid": 16,
+            "dpt": {
+                "main": 1,
+                "sub": 1
+            },
+            "communication_object_ids": [],
+            "description": "Livingroom LivingroomLight"
+        }
+    },
+    "locations": {
+        "Minimal-Example": {
+            "type": "Building",
+            "identifier": "P-05C0-0_BP-1",
+            "name": "Minimal-Example",
+            "usage_id": null,
+            "usage_text": "",
+            "number": "",
+            "description": "",
+            "project_uid": 9,
+            "devices": [],
+            "spaces": {
+                "Livingroom": {
+                    "type": "Room",
+                    "identifier": "P-05C0-0_BP-2",
+                    "name": "Livingroom",
+                    "usage_id": "SU-4",
+                    "usage_text": "Wohnzimmer",
+                    "number": "",
+                    "description": "",
+                    "project_uid": 10,
+                    "devices": [],
+                    "spaces": {},
+                    "functions": [
+                        {
+                            "identifier": "P-05C0-0_F-1",
+                            "name": "LivingroomLight",
+                            "function_type": "FT-1",
+                            "usage_text": "switchable light",
+                            "project_uid": 11,
+                            "group_addresses": [
+                                {
+                                    "identifier": "P-05C0-0_GF-1",
```

Comment:
What's the use case of the identifier? Isn't the `address` enough? 

We also don't have the `identifier` in the `group_adressess` and I don't think we need them or?